### PR TITLE
Psp-3984 cleanup frontend to use Api_Property. Remove references to old IApiPropertyModel. Introduce a hook to facilitate loading composed properties.

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: source/frontend/src/components/maps/leaflet/PointClusterer.tsx [hunk 1]
```diff
@@ -339,22 +296,18 @@ export const PointClusterer: React.FC<PointClustererProps> = ({
...
+                      cluster.properties,
+                      latitude,
+                      longitude,
+                    );
+                    convertedProperty && onMarkerClick(convertedProperty); //open information slideout
+                    setCurrentSelected(convertedProperty);
+
                     selectedPropertyContext.setPropertyInfo(convertedProperty);
+                  } catch (err) {
+                    console.log(err);
```
Comment (FuriousLlama) [inline source/frontend/src/components/maps/leaflet/PointClusterer.tsx:310]: should this be console.error?

File: source/frontend/src/features/mapSideBar/MotiInventoryContainer.test.tsx [hunk 2]
```diff
@@ -76,7 +76,7 @@ describe('MotiInventoryContainer component', () => {
 
   it('shows the property information tab for inventory properties', async () => {
     mockAxios.onPost().reply(200, {});
-    mockAxios.onGet(new RegExp('/properties/*')).reply(200, { id: 9212434 });
+    mockAxios.onGet(new RegExp('/properties/concept/*')).reply(200, { id: 9212434 });
```
Comment (FuriousLlama) [inline source/frontend/src/features/mapSideBar/MotiInventoryContainer.test.tsx:79]: if we are cleaning up the properties API we could also remove the '/concept' part for properties given we would not need that differentiation anymore

File: source/frontend/src/models/api/Property.ts [hunk 3]
```diff
@@ -43,10 +43,12 @@ export interface Api_Property extends Api_ConcurrentVersion, Api_AuditFields {
 
   isSensitive?: boolean;
   isVisibleToOtherAgencies?: boolean;
+  isProvincialPublicHwy?: boolean;
```
Comment (FuriousLlama) [inline source/frontend/src/models/api/Property.ts:46]: Good to have the complete set of members on the front end too

File: source/frontend/src/models/api/Property.ts [hunk 4]
```diff
@@ -43,10 +43,12 @@ export interface Api_Property extends Api_ConcurrentVersion, Api_AuditFields {
...
   isOwned?: boolean;
   isPropertyOfInterest?: boolean;
   pphStatusTypeCode?: string;
   isRwyBeltDomPatent?: boolean;
+  isALR?: boolean;
```
Comment (FuriousLlama) [inline source/frontend/src/models/api/Property.ts:51]: I cant find this field on the backend model. Lets keep the Api_* models a 1:1 to the backend

**Review by FuriousLlama — CHANGES_REQUESTED** [review]
Cannot find 'isALR' on the backend model. Is that a calculated field?

**Comment by asanchezr** [comment]
> Cannot find 'isALR' on the backend model. Is that a calculated field?

Yes, this is populated from one of the map layer queries

File: source/frontend/src/components/maps/leaflet/PointClusterer.tsx (hunk 1 above)
Comment (devinleighsmith) [reply source/frontend/src/components/maps/leaflet/PointClusterer.tsx:310]: it shouldn't be here at all really, was added for debugging purposes. Will remove.

**Comment by devinleighsmith** [comment]
@FuriousLlama anything else on this one or can you clear your changes requested flag?

**Review by FuriousLlama — APPROVED** [review]

**Pull request merged**